### PR TITLE
Add some overlooked prettier changes 💄🔍

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

Tidies up some things I missed in #9424:

- removes some prettier config stuff from DCR and AR
- remove the npm-script for running prettier in DCR (it's run automatically in commit hooks, and can be run manually in DCR with `make fix`
- updates the CI prettier check to run across the repo, and then disables it so CI passes until #9425 is merged

## Why?

I missed these things previously

### CODE REVIEW

File: apps-rendering/package.json [hunk 1]
```diff
@@ -91,7 +90,6 @@
 		"eslint-plugin-eslint-comments": "^3.2.0",
 		"eslint-plugin-import": "^2.26.0",
 		"eslint-plugin-jsx-a11y": "^6.7.1",
-		"eslint-plugin-prettier": "^4.2.1",
```
Comment (sndrs) [inline apps-rendering/package.json:94]: this wasn't actually being used afaict

Comment (mxdvl) [reply apps-rendering/package.json:94]: Indeed, it seems like we use [`eslint-config-prettier`](https://github.com/prettier/eslint-config-prettier#installation)

File: package.json [hunk 2]
```diff
@@ -27,10 +27,12 @@
...
+		"lint-staged": "lint-staged",
+		"prettier:check": "prettier --ignore-unknown --check --cache .",
+		"prettier:write": "prettier --ignore-unknown --write --cache ."
 	},
 	"lint-staged": {
-		"*": "prettier --ignore-unknown --write"
+		"*": "prettier --ignore-unknown --write --cache"
 	},
 	"dependencies": {
 		"@guardian/prettier": "^4.0.0",
```
Comment (mxdvl) [inline package.json:38]: Shall we move to v5?

**Review by mxdvl — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (sndrs) [reply package.json:38]: oh good spot thanks, yes agree

File: yarn.lock [hunk 3]
```diff
@@ -15122,7 +15117,7 @@ prebid.js@guardian/prebid.js#0b4cccd:
     core-js "^3.13.0"
     core-js-pure "^3.13.0"
     criteo-direct-rsa-validate "^1.1.0"
-    crypto-js "^4.2.0"
+    crypto-js "^3.3.0"
```
[the remark this answers is not in the document]
Comment (mxdvl) [reply yarn.lock:15125]: It seems like this made its way back in…

**Pull request merged**